### ISSUE TITLE
四，tf.nn.softmax_cross_entropy_with_logits(output_layer, y)

### ISSUE THREAD

**Issue by zdpau** [zdpau/-TensorFlow-#4]
解决方法：tf.nn.softmax_cross_entropy_with_logits(**labels=**output_layer, **logits=**y)
